noir-projects/noir-contracts/contracts/test_contract/src/main.nr

```diff
@@ -130,6 +130,44 @@ contract Test {
         emit_unencrypted_log_from_private(&mut context, opt_notes[1].unwrap().value);
     }
 
+    pub fn filter_that_creates_sparse_array(
```

Comment:
Could you create an issue with the recommendation from @nventuro and point here and above create_and_get_many_notes_with_filter to nuke this once we have proper noir testing?